src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -351,14 +351,17 @@ private async Task UnsynchronizedStartHostAsync(ScriptHostStartupOperation activ
 
                 ActiveHost = localHost;
 
-                // Forward initial startup logs to AppInsights/OpenTelemetry.
-                // These are not tracked by the AppInsights and OpenTelemetry logger provider as these are added in the script host.
-                var loggerProviders = ActiveHost.Services.GetServices<ILoggerProvider>();
-                var deferredLogProvider = ActiveHost.Services.GetService<DeferredLoggerProvider>();
-                if (deferredLogProvider is not null)
-                {
-                    var selectedProviders = loggerProviders.Where(provider => provider is ApplicationInsightsLoggerProvider or OpenTelemetryLoggerProvider).ToArray();
-                    deferredLogProvider.ProcessBufferedLogs(selectedProviders);
+                if (!FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagDisableWebHostLogForwarding, _environment))
```

Comment:
I think we should rename the FeatureFlags.IsEnabled to 'IsConfigured', 'IsSet' or 'Exists' for better readability.

Reply:
I kind of agree with you, this is confusing. It's used at so many places, let's talk about this in standup.